osc-server/src/main/java/org/osc/core/broker/util/network/NetworkSettingsApi.java

```diff
@@ -148,4 +90,52 @@ public NetworkSettingsDto getNetworkSettings() {
         return dns;
     }
 
-}
+    private String getNetMask(String ipAddr) {
```

Comment:
check whether any build-in java methods for netmask.

Reply:
There is no direct Java API to get the netmask.So,need to get as custom way.